code/drasil-example/Drasil/GlassBR/Body.hs

```diff
@@ -330,22 +326,79 @@ org_of_doc_intro_end = foldl (+:+) EmptyS [(at_start' $ the dataDefn),
 
 {--GENERAL SYSTEM DESCRIPTION--}
 
+general_system_description :: Section
+general_system_description = assembler gLassBR gbSymbMap generalSystemDescriptionSect
+  [sysContext, userCharacteristicSect, systemConstraintSect]
+
```

Comment:
Some of this code is not being used and should be removed so we're not polluting the examples. Now that you're not using `Verbatim`, you can get rid of the `SubSec`s if they're not explicitly used in the `GSDSec` constructor above.